packages/trader/src/sass/app/_common/layout/trader-layouts.scss

```diff
@@ -468,6 +468,9 @@ $FLOATING_HEADER_HEIGHT: 41px;
         .sc-categorical-display {
             height: calc(100% - 8px) !important;
         }
+        .chartContainer {
+            width: calc(100% - 0.8rem);
```

Comment:
`padding: 0 0.8rem` will do the work. `width` is only affecting one side of the chart but design shows padding from both sides.